src/test/resources/features/UpdateRouteVersion.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Update RouteVersion
+  In order to use the app
+  As a user
+  I want to update RouteVersion and check throw errors
+
+  Background:
+    Given There is a registered user with username "username" and password "password" and email "user@domain.com"
+    Given There is a route with a title "testRoute", description "route description", type "Running" and a creationDate "2023-10-25T17:27:00Z" by user username "username"
+    Given I create a new correct RouteVersion whit a Route title "testRoute"
+
+  Scenario: Update RouteVersion
+    Given I login as "username" with password "password"
+    When I update a creationDate RouteVersion whit a Route title "testRoute"
+    Then The response code is 403
```

Comment:
Right now any kind of update using PATCH is blocked due to the WebSecurityConfig `denyAll`, but not if the change is done using POST. It might be worth blocking it too if no changes to route versions are allowed once they are created